### DISCUSSION TITLE
Tell market Ladyes to walk on our ways

### DISCUSSION THREAD

**Discussion by BraianCAESAR** [Keriew/augustus#273] [Ideas]
Would be very nice if we could choose the path ladyes walk, prefects walk, etc. 

My prefects always let the city burn down while the engenniers walk all over the city doing well their job. The brown citizens could walk more too to find peoples do work on.

**Comment by crudelios** [comment]
This is a core mechanic of Caesar 3, so I don't think we should change it actually. We've been tweaking some things though, for example there is now a setting to disable buildings needing to be close to homes in order to get workers.

**Reply by BraianCAESAR** [reply]
I got ti, but totally random mechanic, they have no path to go with. Sometimes they go dumb paths for no reasons, like prefects walking and backing same road 3-4 times then backing prefecture.

**Reply by BraianCAESAR** [reply]
More peoples working on Prefectures and 2 prefects same time would be better maybe?

**Reply by crudelios** [reply]
Well, the whole idea of the game is to avoid creating too many paths. The game says so itself in the initial assignment for the first mission.

It's not that it is technically impossible for us to change that behaviour, but by doing that, we would be ditching away the whole gameplay concept behind C3, which attrated players (myself included) in the first place.

Right now, regarding your prefects problem, you have the following solutions:

- Avoid intersections
- Use roadblocks
- Create more prefectures to increase coverage

To me and many others, part of the game's magic is creating a nice, stable city despite all the apparent randomness.

With that being said, we may still improve the way random walkers move about in the future, but it's not on our immediate plans to do it as it would be a very drastic change that would require lots of community feedback and balancing to get right.

We're also having discussions about adding two walkers for some buildings, like markets and some entertainment buildings, but those ideas are still in the early stages too.

**Reply by Italicus66** [reply]
crudelios wrote: "We're also having discussions about adding two walkers for some buildings, like markets and some entertainment buildings, but those ideas are still in the early stages too."

For me it would be appreciated... :-D   My market ladys are often not able to take enough potterys to the houses in spite of the fact around are full storehouses of pottery.
I guess I'm just a bad governor... :-D
Maybe one could bring it as an option in the game-settings?

By the way: Very much thank you for the devs for making Augustus! :-)
